src/WpfMath/WpfMath.csproj

```diff
@@ -18,6 +20,6 @@
         <Resource Include="Fonts\cmsy10.ttf" />
     </ItemGroup>
     <ItemGroup>
-      <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+      <PackageReference Condition="'$(TargetFramework)'=='net461'" Include="System.ValueTuple" Version="4.4.0" />
```

Comment:
I understand why you've added a condition here, but why downgrade it to 4.4.0?

Reply:
Foq and FSharp.Core.Fluent-4.0 referenced `System.ValueTuple` 4.4.0 which will cause conflicts with 4.5.0 (during build and it will throw a warning)